Tests/Fixtures/include_test.yml

```diff
@@ -19,3 +26,5 @@ targets:
     name: IncludedTargetNew
     platform: tvOS
     sources:REPLACE: NewSource
+    dependencies:
+      - package: Yams
```

Comment:
I added to check the behavior of dependency because it is motivated one what I want by this changing. It is my preference so I can remove it, What do you think?